gc/base/GCExtensionsBase.cpp

```diff
@@ -130,6 +130,12 @@ MM_GCExtensionsBase::initialize(MM_EnvironmentBase* env)
 	gcmetadataPageSize = pageSizes[0];
 	gcmetadataPageFlags = OMRPORT_VMEM_PAGE_FLAG_NOT_USED;
 
+	heapExpansionGCRatioThreshold._valueSpecified = 13;
+	heapContractionGCRatioThreshold._valueSpecified = 5;
+
+	heapExpansionGCTimeThreshold = 13; //To be removed
+	heapContractionGCTimeThreshold = 5; //To be removed
```

Comment:
it's initialized in constructor (which you will remove later), so this is not necessary

Reply:
You're right, removing these 2 lines now